core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaLevelEventTranslator.java

```diff
@@ -346,6 +382,57 @@ public void translate(GeyserSession session, ClientboundLevelEventPacket packet)
                 session.sendUpstreamPacket(soundEventPacket);
                 return;
             }
+            // Half of these are listed in Cloudburst as Trail instead of Trial for some reason
```

Comment:
btw - I did PR the "trail"-typos to protocol (https://github.com/CloudburstMC/Protocol/pull/238), feel free to [bump the cloudburst protocol library here](https://github.com/GeyserMC/Geyser/blob/master/gradle/libs.versions.toml#L13) to the latest version found [here](https://repo.opencollab.dev/#/maven-snapshots/org/cloudburstmc/protocol/bedrock-codec/3.0.0.Beta2-SNAPSHOT), then the names should be fixed

Reply:
I was trying to figure out how to do that because i saw you fixed it, thanks for showing me how

Reply:
updated :)